backend/src/ml_space_lambda/enums.py

```diff
@@ -101,6 +103,7 @@ class Permission(Enum):
     PROJECT_OWNER = "MO"
     ADMIN = "PMO"
     ACTING_PMO = "actingPMO"
+    GROUP_OWNER = "GO"
```

Comment:
I think we need to remove Group owner. For right now it seems like we want to lock down Group ownership to just admins (according to Cindy's notes), so I don't think this new role will give us anything.

Reply:
Actually you know what, I feel like it'll be easier to just give an admin `GO` permissions when they create a group than to remove this and rip out all of the GroupOwner stuff, only to add it back in later if we want to support this for non-admins.

Reply:
I was thinking we can add it right now and keep it similar to what we do for projects and then rip it out if we end up not wanting it before we call it feature complete.